.gitignore

```diff
@@ -72,5 +72,8 @@ sonar-project.properties
 #Helm
 **/Chart.lock
 
+#Everything else:
+*.DS_Store
```

Comment:
I'm super-duper surprised this isn't in .gitignore by default in our template projects! Might be worth raising a PR on those to add this 😃